src/site/navigation/facility_sidebar_nav.drupal.liquid

```diff
@@ -0,0 +1,40 @@
+            <nav id="va-detailpage-sidebar" class="usa-width-one-fourth va-sidebarnav">
+    <div>
+        <button class="va-btn-close-icon va-sidebarnav-close" type="button" aria-label="Close this menu"></button>
+        <div class="left-side-nav-title">
+            <h4>{{ facilitySidebar.name }}</h4>
+        </div>
+        <ul class="usa-accordion">
+            {% for link in facilitySidebar.links %}
+                <li>
+                    <button class="usa-accordion-button" aria-expanded="false" aria-controls="a{{ forloop.index }}">{{ link.label }}</button>
```

Comment:
I think `aria-expanded` should be set to `link.expanded`

Reply:
link.expanded is an indicator on the drupal side to check if items have children. It doesn't correspond to the state of the accordion on the front end, unfortunately. added another way to check for whether they should be expanded tho.